packages/core/src/external/carequality/saml/security/security-header.ts

```diff
@@ -0,0 +1,174 @@
+import * as pkijs from "pkijs";
+import * as asn1js from "asn1js";
+import { arrayBufferToString, toBase64 } from "pvutils";
+
+export const security_header_timestamp_id = "TS-7c229e85-d62b-471e-9112-a49d1c365004";
```

Comment:
We need to have reference IDs in our SAML linking the signature and the thing being signed. These could be anything,  but I just picked some random uuids and stuck with them.

Reply:
No it can - they aren't secrets in any way. They are just pointers within the SAML telling you where to find the signature inside ot